src/AdvancedMH.jl

```diff
@@ -160,4 +160,8 @@ include("mh-core.jl")
 include("emcee.jl")
 include("MALA.jl")
 
+include("RobustAdaptiveMetropolis.jl")
+using .RobustAdaptiveMetropolis
+export RAM
```

Comment:
It's the name used by the paper itself, so figured this was the way to go :grimacing:

Reply:
Hmmm, but the authors (and readers) already know where it's explained 😄 IMO `RWMH` is also not a great name (`RandomWalkMH` doesn't seem too bad?), so I thought maybe one could make it more descriptive right away this time.

Reply:
Hmm, so I agree with you that it's not the most clear name, _however_ I'm preferential to using the algorithm name as it appears in the literature, even if the chosen name isn't the best 😕 

It will be a bit annoying if people read about "RAM" and then we don't have this algorithm because it's under a different name..

Reply:
Renamed to `RobustAdaptiveMetropolis` to remove any ambiguity